assignment-1/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  backend "s3" {
+    bucket = "cap-dev-dnyaneshwar"
+    key    = "stateBucket"
+    region = "us-east-2"
+  }
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
```

Comment:
exact version mentioning is good practice eg 5.0.0, also terraform version is not given